src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.builds

```diff
@@ -31,7 +31,7 @@
       <OSGroup>Windows_NT</OSGroup>
       <Platform>amd64</Platform>
     </Project>
-    <Project Include="win7\runtime.native.System.IO.Compression.pkgproj">
+    <Project Include="win8\runtime.native.System.IO.Compression.pkgproj">
```

Comment:
Perhaps the diff isn't showing it correctly but shouldn't you need to delete the win7 folder?